lib/topological_inventory/ansible_tower/targeted_refresh/service_instance.rb

```diff
@@ -34,7 +36,10 @@ def refresh
           logger.info_ext(operation, "Connection set successfully")
 
           # Get all tasks
-          tasks, refresh_state_uuid = {}, SecureRandom.uuid
+          tasks, @refresh_state_uuid = {}, SecureRandom.uuid
```

Comment:
If we're going to make `refresh_state_uuid` an instance variable - maybe we don't need to pass it around between functions anymore?  Mostly talking about the `refresh_part_*` and `async_save_inventory`/`async_collecting_finished` methods.

Reply:
The async methods receives them from AsyncReceiver (shared with full on-premise refresh), so the param has to be there, using `@` var if there is param of the same name will be confusing. 
Not sure what do you mean by the `refresh_part_*` thing

Reply:
@lindgrenj6 btw do you want I refactor it? Just saying the complete retesting is quite time-consuming

Reply:
Well I guess that is my question - I'm curious why we need to pass `@refresh_state_uuid` to the `refresh_part` method - since it is an instance variable now can't we just access it via `@refresh_state_uuid` instead of using the parameter?

Reply:
we can use it directly, so ok, I'll change it